README.md

```diff
@@ -82,7 +83,12 @@ Write the BNF (not EBNF) description for this language.
 
 ## A2.1
 
-«replace this with your answer»
+<sentence> ::= "the" <adjectives> <noun> <verb> <adverb>
+<adjectives> ::= <adjectives> <adjective> | <adjective>
+<adjective> ::= "lazy" | "smelly" | ""
+<noun> ::= "dog" | "cat"
+<verb> = "ate" | "ran"
+<adverb> = "slowly" | "noisily" | ""
```

Comment:
No deduction, but my spidey sense is tingling with the lines:

~~~
<adjectives> ::= <adjectives> <adjective> | <adjective>
<adjective> ::= "lazy" | "smelly" | ""
~~~

I wonder if this might loop forever because <adjectives> can repeatedly call <adjective> matching nothing. I think it would be safer to have

~~~
  <adjectives> ::= "" | <adjectives> <adjective>
  <adjective> ::= "lazy" | "smelly"
~~~